src/redis.go

```diff
@@ -136,7 +137,12 @@ func createIntegration() (*integration.Integration, error) {
 
 func entity(i *integration.Integration, args *argumentList) (*integration.Entity, error) {
 	if args.RemoteMonitoring {
-		n := fmt.Sprintf("%s:%d", args.Hostname, args.Port)
+		var n string
+		if args.UseUnixSocket && args.UnixSocketPath != "" {
+			n = fmt.Sprintf("%s:%s", args.Hostname, args.UnixSocketPath)
```

Comment:
Why the entity name is `hostname:socket` and not `hostname:port:socket`?

I guess that it is enough to avoid collisions, but isn't better to concatenate the new data rather then modifying it?

Reply:
either you call the integration using unix sockets or using ip:port. The default hostname is localhost, whose value will be replaced by the agent with the real hostname. The port will probably be the default one, which won't make the entities unique.